aiida_test_cache/__init__.py

```diff
@@ -1,3 +1 @@
 """A pytest plugin for testing AiiDA plugins."""
-
-__version__ = '0.1.0.dev1'
```

Comment:
I don't think the `__version__` attribute is very useful for this package and it complicates package management so I am removing it. Dissenters will be pointed to a lengthy discussion thread on CPython forum full of people arguing about this. :-D

Reply:
Please point me to the lengthy discussion, would be a nice read material after dinner.

Reply:
:rofl: 

https://discuss.python.org/t/brainstorming-eliminating-dynamic-metadata/71405

Here you go. :-)